src/test-helpers.tsx

```diff
@@ -650,3 +651,12 @@ export const createFakeRef = (
     },
   };
 };
+
+export const createStoreWithVersion = (
```

Comment:
Should we file an issue to use this helper function everywhere else?

Reply:
Yeah, good idea. I did a quick search and it can be used in a lot of places.